ui/component/settingContent/view.jsx

```diff
@@ -119,6 +121,15 @@ export default function SettingContent(props: Props) {
               />
             </SettingsRow>
 
+            <SettingsRow title={__('Hide Personal Recommendations')} subtitle={__(HELP.HIDE_FYP)}>
```

Comment:
This will be a good time to start a premium framework on settings page - only show this if premium. If it's easy, also only show if you have recommendations (someone may be confused if they don't see them).

Reply:
Updated with Premium tag + gray out when count is 0 or not a member